app/build.gradle

```diff
@@ -64,7 +64,10 @@ dependencies {
       'com.google.dagger:dagger:2.24',
       "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
       "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.2.1",
-      "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.2.1"
+      "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.2.1",
+      'androidx.recyclerview:recyclerview:1.0.0',
+      'com.github.bumptech.glide:glide:4.9.0',
+      'org.jsoup:jsoup:1.12.1',
```

Comment:
What is this dependency for?

Reply:
This was used to parse html content. But now I used some different solution so I have removed this dependency